docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -202,8 +202,9 @@ Methods
 |                             |                                 | Categoricals are not implemented | label or Series from the current DataFrame;        |
 |                             |                                 | yet                              | otherwise ``N``;                                   |
 |                             |                                 |                                  | Note that supported functions are agg, count,      |
-|                             |                                 |                                  | cumcount, cummax, cummin, cumsum, max, mean,       |
-|                             |                                 |                                  | median, min, quantile, shift, std, sum, and var.   |
+|                             |                                 |                                  | cumcount, cummax, cummin, cumsum, first, last,     |
```

Comment:
`first`, `last`, and `size` were already implemented but were accidentally missed here in prior PRs?

Reply:
Yup - seems to be that way!